unitstatushandler/unitstatushandler.go

```diff
@@ -646,6 +646,12 @@ func (instance *Instance) sendCurrentStatus() {
 		return
 	}
 
+	for i := len(instance.instanceStatuses) - 1; i >= 0; i-- {
+		if instance.instanceStatuses[i].RunState == "" {
```

Comment:
I don't understand where empty `RunState` came from? I don't see this path.

Reply:
Use instead of empty activating service state